dbt/include/dremio/macros/adapters/metadata.sql

```diff
@@ -140,7 +140,7 @@ limitations under the License.*/
         + (('.' + schema) if schema != 'no_schema' else '') -%}
   {% call statement('list_relations_without_caching', fetch_result=True) -%}
 
-    {%- if var('dremio:reflections_enabled', true) -%}
+    {%- if var('dremio:reflections_enabled', false) -%}

```

Comment:
The logic here seems like it'll select from sys.reflections if dremio:reflections_enabled is set to false?

Reply:
That's what I thought initially as well, the second argument in var() is actually `default`, which is the default value for the variable if one is not explicitly defined.
https://docs.getdbt.com/reference/dbt-jinja-functions/var#variable-default-values

Reply:
Can we use a keyword argument, i.e default=false?

Reply:
Added in commit.